packages/tools/heft-rig/tsconfig.playwright.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "baseUrl": "packages",
+    "paths": {
+      "@kadena/*": ["packages/tools/e2e-components"] // This mapping is relative to "baseUrl".
+    }
+  }
+}
```

Comment:
Why do we need this path mapping here? Mapping like looks odd, since we have packages with the same name. Maybe they should just be (dev/peer) dependencies?

Reply:
I could not get that to work fully, I'll revisit and ask for help if I get stuck.